docs/src/components/swatch-grid.tsx

```diff
@@ -0,0 +1,147 @@
+import {Box} from '@primer/components'
+import get from 'lodash.get'
+import React from 'react'
+import colors from '../../../dist/js/colors_v2'
+import {getFullName} from '../../../script/lib/variable-collection'
+import {useColorTheme} from './color-theme-context'
+
+export function SwatchGrid({names}: {names: string[]}) {
+  return (
+    <Box
+      sx={{
+        display: 'grid',
+        gridGap: 3,
+        gridTemplateColumns: 'repeat(auto-fill, minmax(200px, 1fr))'
+      }}
+    >
+      {names.map(name => (
+        <Swatch name={name} />
+      ))}
+    </Box>
+  )
+}
+
+function Swatch({name}: {name: string}) {
+  const [colorTheme] = useColorTheme()
+  const value = React.useMemo(() => get(colors[colorTheme], name), [name, colorTheme])
+  const variant = React.useMemo(() => chooseVariant(name), [name])
+
+  return (
+    <Box>
+      {variant === 'fg' && <FgPreview name={name} value={value} />}
+      {variant === 'border' && <BorderPreview name={name} value={value} />}
+      {variant === 'shadow' && <ShadowPreview name={name} value={value} />}
+      {variant === 'default' && <DefaultPreview name={name} value={value} />}
+      <Box sx={{fontFamily: 'mono', fontSize: 1, mt: 2}}>{name}</Box>
+      <Box sx={{fontFamily: 'mono', fontSize: 0, color: 'text.gray'}}>{`--${getFullName(
```

Comment:
I wonder if we should wrap this in `var(--color-fg-default)` to make it more clear it's css? So it would look like this. 
![image](https://user-images.githubusercontent.com/54012/132063893-509e2e3e-312b-4c39-95bc-d0d458ce7c6d.png)

Reply:
Sounds good to me 👍